nilearn/tests/test_extmath.py

```diff
@@ -8,11 +8,16 @@
 
 from nilearn._utils.extmath import fast_abs_percentile, is_spd
 
+from sklearn.datasets import load_iris
+from ..image import mean_img
+from ..decoding.tests.test_same_api import to_niimgs
+from ..plotting.find_cuts import find_xyz_cut_coords
+
 
 def test_fast_abs_percentile():
     data = np.arange(1, 100)
```

Comment:
Can you shuffle it? This is the reason the test were passing before with the completely broken implementation of fast_abs_percentile.